src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  flex-direction: column;
```

Comment:
The `flex-direction: column;` property is unnecessary here because `.stars__star` is set to `display: block;`, which does not support flex properties. Consider removing this line.